lib/fastlane/plugin/emerge/actions/emerge_action.rb

```diff
@@ -172,65 +175,70 @@ def self.details
+                                       type: String),
           FastlaneCore::ConfigItem.new(key: :base_sha,
-                               description: "The git SHA of the base build",
-                                  optional: true,
-                                      type: String),
+                                       description: "The git SHA of the base build",
+                                       optional: true,
+                                       type: String),
           FastlaneCore::ConfigItem.new(key: :build_id,
-                               description: "A string to identify this build",
-                                deprecated: "Replaced by `sha`",
-                                  optional: true,
-                                      type: String),
+                                       description: "A string to identify this build",
+                                       deprecated: "Replaced by `sha`",
+                                       optional: true,
+                                       type: String),
           FastlaneCore::ConfigItem.new(key: :base_build_id,
-                               description: "Id of the build to compare with this upload",
-                                deprecated: "Replaced by `base_sha`",
-                                  optional: true,
-                                      type: String),
+                                       description: "Id of the build to compare with this upload",
+                                       deprecated: "Replaced by `base_sha`",
+                                       optional: true,
+                                       type: String),
           FastlaneCore::ConfigItem.new(key: :repo_name,
-                               description: "Full name of the respository this upload was triggered from. For example: EmergeTools/Emerge",
-                                  optional: true,
-                                      type: String),
+                                       description: "Full name of the respository this upload was triggered from. For example: EmergeTools/Emerge",
+                                       optional: true,
+                                       type: String),
           FastlaneCore::ConfigItem.new(key: :gitlab_project_id,
-                               description: "Id of the gitlab project this upload was triggered from",
-                                  optional: true,
-                                      type: Integer),
+                                       description: "Id of the gitlab project this upload was triggered from",
+                                       optional: true,
+                                       type: Integer),
+          FastlaneCore::ConfigItem.new(key: :tag,
+                                       description: "String to label the build. Useful for grouping builds together in our dashboard, like debug, release, or pull-request. Defaults to development",
```

Comment:
Should we update the default here to be "default"? Also maybe change the description to not include debug/release since those cause confusion, and just say `default` or `pull-request`?